index.html

```diff
@@ -34,10 +34,11 @@ <h2 class="tel"><a href="tel:+380990671174">+38(099) 067-11-74</a></h2>
         </div>
     </header>
     <main>
-        <div class="work">
-            <h3 class="text-about-us">
-                Замовте майстра на всі руки, якому можна довіряти</h3>
-        </div>
+        <div class="hero-section">
+            <h3 class="motto">
+              Замовте майстра на всі руки, якому можна довіряти!<br>
+              Ваше задоволення - наш пріоритет.</h3>
```

Comment:
Hehe, here is the tricky part! Please use `&mdash;` instead of hyphen (`-`). And please investigate how to use dashes, hyphens, minuses in text: https://hyperhost.ua/info/ru/tire-minus-defis-v-cem-raznica